project/Dependencies.scala

```diff
@@ -23,6 +25,7 @@ object Version {
 }
 
 object Library {
+  val playLogback         = "com.typesafe.play"        %% "play-logback"             % Version.play
   val playJdbcApi         = "com.typesafe.play"        %% "play-jdbc-api"            % Version.play
```

Comment:
Are these dependencies necessary ?

Reply:
For tests, yeah. Unless we decide to remove `ColoredLevel`:

https://github.com/playframework/play-slick/blob/master/src/core/src/test/resources/logback-test.xml#L3

Reply:
Oh, play 2.7.0 changed the transitive deps.